.circleci/config.yml

```diff
@@ -72,9 +72,7 @@ workflows:
     jobs:
       - ios
       - macos
-      - carthage:
-          requires:
```

Comment:
I believe removing this may allow Carthage to run before iOS (I think the jobs start randomly and the only thing that prevents it is "requires"). Also, Carthage takes the longest, so if the iOS build fails, no reason to run Carthage because it will likely fail. Better to find out quickly that iOS failed than wait until after Carthage

Reply:
Yeah I meant to explain myself here. My reasoning was that the iOS tests still fail randomly; that the tests fail doesn't mean the binary isn't good. And if there's been an issue with the Carthage build, working on getting it green is a crapshoot. 

It's more of a hack to deal with what we've got, than the right solution.